weekly-mission(week2)/src/features/signUp.js

```diff
@@ -0,0 +1,21 @@
+import { addErrorMsgToMismatchConfirm } from "../view/addErrorMessageToInput.js";
+
+export const signUpHandler = (e) => {
+  e.preventDefault();
+  addErrorMsgToMismatchConfirm();
+
+  const inputs = document.querySelectorAll('.input-profile', e.target);
+  inputs.forEach((input) => {
+    input.dispatchEvent(new Event('blur'));
+  });
+
+  const hasError = [...inputs].some((input) => {
+    return input.classList.contains('js-input-profile-error');
+  });
+  const hasBlankInput = [...inputs].some((input) => input.value === '')
+
+  if (hasError) return;
+  if (hasBlankInput) return;
```

Comment:
우선은 이렇게 리팩토링해주는 건 좋을 것 같아요!
```suggestion
  if (hasError || hasBlankInput) return;
```

Reply:
그리고 이 경우에는 개인적으로는 중괄호를 안써도 괜찮다고 생각하는데요! 그래도 저는 개인적으로 중괄호를 써주기는 해요.
이 부분은 인석님의 취향으로 해주시면 좋을 것 같아요!